environments/icds-cas/inventory/riakcs.csv

```diff
@@ -1,11 +1,11 @@
-hostname,host_address,group 1,group 2,group 3,var: hostname,L.var: devices,L.var: partitions
-riak0,100.71.184.33,riakcs,,stanchion,MUMGCCWCDPRDR000,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak1,100.71.184.43,riakcs,,,MUMGCCWCDPRDR001,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak2,100.71.184.13,riakcs,,,MUMGCCWCDPRDR002,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak3,100.71.184.44,riakcs,,,MUMGCCWCDPRDR003,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak4,100.71.184.50,riakcs,,,MUMGCCWCDPRDR004,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak5,100.71.184.37,riakcs,,,MUMGCCWCDPRDR005,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak6,100.71.184.48,riakcs,,,MUMGCCWCDPRDR006,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak7,100.71.184.40,riakcs,,,MUMGCCWCDPRDR007,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak8,100.71.184.46,riakcs,,,MUMGCCWCDPRDR008,"[""/dev/vdb""]","[""/dev/vdb1""]"
-riak9,100.71.184.35,riakcs,,,MUMGCCWCDPRDR009,"[""/dev/vdb""]","[""/dev/vdb1""]"
+hostname,host_address,group 1,group 2,group 3,var: hostname,L.var: devices,L.var: partitions,var: encrypted_root
+riak0,100.71.184.33,riakcs,,stanchion,MUMGCCWCDPRDR000,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data
+riak1,100.71.184.43,riakcs,,,MUMGCCWCDPRDR001,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data
+riak2,100.71.184.13,riakcs,,,MUMGCCWCDPRDR002,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data1
+riak3,100.71.184.44,riakcs,,,MUMGCCWCDPRDR003,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data1
+riak4,100.71.184.50,riakcs,,,MUMGCCWCDPRDR004,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data1
+riak5,100.71.184.37,riakcs,,,MUMGCCWCDPRDR005,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data1
+riak6,100.71.184.48,riakcs,,,MUMGCCWCDPRDR006,"[""/dev/vdb""]","[""/dev/vdb1""]",/opt/data1
```

Comment:
Why did these get something other than `/opt/data`?

Reply:
We've had some issues with the LVM setup. Somehow the LVM group got deleted on all the DB VMs so we've had to re-create them all. These ones had data from NIC on them which we didn't want to loose so we added new disks to copy the data to before removing the old ones.